Tools/clinic/clinic.py

```diff
@@ -432,10 +432,11 @@ class FormatCounterFormatter(string.Formatter):
          the counts dict would now look like
          {'a': 2, 'b': 1, 'c': 1}
     """
-    def __init__(self):
-        self.counts = collections.Counter()
+    def __init__(self) -> None:
+        self.counts: dict[str, int] = collections.Counter()
```

Comment:
I would go for

```suggestion
        self.counts = collections.Counter[str]()
```

Or

```suggestion
        self.counts: collections.Counter[str] = collections.Counter()
```

Reply:
Oh, TIL!